src/foam/u2/filter/BooleanFilterView.js

```diff
@@ -0,0 +1,128 @@
+/**
+* @license
+* Copyright 2019 The FOAM Authors. All Rights Reserved.
+* http://www.apache.org/licenses/LICENSE-2.0
+*/
+
+foam.CLASS({
+  package: 'foam.u2.filter',
+  name: 'BooleanFilterView',
+  extends: 'foam.u2.Controller',
+
+  documentation: `
+    A simple Boolean filter that allows the user to pick between true or false.
+  `,
+
+  implements: [
+    'foam.mlang.Expressions'
+  ],
+
+  requires: [
+    'foam.u2.CheckBox'
+  ],
+
+  css: `
+    ^container {
+      display: flex;
+      align-items: center;
+      padding: 4px 16px;
+    }
+
+    ^container:hover {
+      cursor: pointer;
+      background-color: #f5f7fa;
+    }
+
+    ^container:first-child {
+      margin-top: 20px;
+    }
+
+    ^container:last-child {
+      margin-bottom: 20px;
+    }
+
+    ^container .foam-u2-md-CheckBox-label {
+      position: relative;
+      margin-top: 0;
+    }
+
+    ^container .foam-u2-md-CheckBox {
+      border-color: #9ba1a6;
+    }
+
+    ^container .foam-u2-md-CheckBox:checked {
+      background-color: #406dea;
+      border-color: #406dea;
+    }
+  `,
+
+  properties: [
+    {
+      name: 'property',
+      documentation: `The property that this view is filtering by. Should be of
+      type Boolean.`,
+      required: true
+    },
+    {
+      class: 'Boolean',
+      name: 'bool1',
+      documentation: `Lets the user pick boolean they want to filter by`,
+      value: false
```

Comment:
Shouldn't the value for bool1 be 'true'? Should you be adding labels and more explicit documentation to both bool1 and bool2 so it is clear what values they represent?